Riot/Modules/Room/RoomViewController.m

```diff
@@ -1615,6 +1612,25 @@ - (void)showRoomCreationModalWithBubbleData:(id<MXKRoomBubbleCellDataStoring>) b
     [self.roomCreationModalCoordinatorBridgePresenter presentFrom:self animated:YES];
 }
 
+- (void)showMemberDetails:(MXRoomMember *)member
+{
+    if (!member)
+    {
+        return;
+    }
+    RoomMemberDetailsViewController *memberViewController = [RoomMemberDetailsViewController roomMemberDetailsViewController];
+    
+    // Set delegate to handle action on member (start chat, mention)
+    memberViewController.delegate = self;
+    memberViewController.enableMention = (self.inputToolbarView != nil);
+    memberViewController.enableVoipCall = NO;
+    
+    [memberViewController displayRoomMember:member withMatrixRoom:self.roomDataSource.room];
+    
+    [self vc_removeBackTitle];
```

Comment:
Maybe we can remove the back title in `viewDidLoad` if we never expect to display the room title as back title for any view controller

Reply:
Makes sense, updated